operator/apis/lifecycle/v1alpha3/keptntaskdefinition_types.go

```diff
@@ -24,12 +24,26 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+const (
+	PYTHON_RUNTIME = "python"
+	//JS_RUNTIME     = "JS"
+)
+
 // KeptnTaskDefinitionSpec defines the desired state of KeptnTaskDefinition
 type KeptnTaskDefinitionSpec struct {
+	// Deprecated
 	// Function contains the definition for the function that is to be executed in KeptnTasks based on
 	// the KeptnTaskDefinitions.
 	// +optional
 	Function *FunctionSpec `json:"function,omitempty"`
+	// Python contains the definition for the python function that is to be executed in KeptnTasks based on
+	//	the KeptnTaskDefinitions.
+	// +optional
+	Python *FunctionSpec `json:"python,omitempty"`
+	// JS contains the definition for the JS function that is to be executed in KeptnTasks based on
+	//	the KeptnTaskDefinitions.
+	// +optional
+	JS *FunctionSpec `json:"js,omitempty"`
```

Comment:
I think it would make more sense to not use the abbreviation here
```suggestion
	JS *FunctionSpec `json:"javascript,omitempty"`
```